plugins/config_manager.py

```diff
@@ -88,6 +91,9 @@ def modify_repo(self):
         modify = {}
         if hasattr(self.opts, 'main_setopts') and self.opts.main_setopts:
             modify = dict(self.opts.main_setopts._get_kwargs())
+        if self.opts.dump_variables:
+            for name, val in self.base.conf.substitutions.items():
+                print("%s = %s" % (name, val))
```

Comment:
* I think there should be no spaces (I think should be quite useful to do `| grep "^key=" | eval`
* value must be quoted (see `shlex.quote()`)